src/game/SpellEffects.cpp

```diff
@@ -1149,6 +1149,13 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
 
                     return;
                 }
+                case 32027:                                 // Expedition Flare
+                {
+                    if (!m_CastItem)                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
```

Comment:
@robinsch I would very much agree with you there; I have no idea why our system is designed so that items are removed before we actually done with them.